src/averell/readers/disco.py

```diff
@@ -1,5 +1,65 @@
+import xml.etree.ElementTree as ETree
+from pathlib import Path
+
+
+def parse_xml(xml_file):
+    """
+    XML TEI poem parser for 'disco' corpus
+    :param xml_file: path of xml file
+    :return: poem dict
+    """
+    tree = ETree.parse(xml_file)
+    root = tree.getroot()
+
+    poem = {}
+    stanza_list = []
+
+    analysis_description = root.find(
+        ".//{http://www.tei-c.org/ns/1.0}metDecl/{http://www.tei-c.org/ns/1.0}p").text
```

Comment:
Maybe we can add `{http://www.tei-c.org/ns/1.0}` and `{http://www.tei-c.org/ns/1.0}` as variables or constants so we don't have them repeated

Reply:
Done